src/app/components/signup/signup.component.css

```diff
@@ -0,0 +1,41 @@
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+}
+input{
+    outline:none;
+    border:none;
+    /* border-radius: 10px; */
+    border-bottom:2px solid gray;
+    width: min(80svw, 900px);
+    /* height: 1rem; */
+    /* width: 100%; */
+    height: 2rem;
+    
+
+}
+form{
+    /* border:2px solid #fff; */
+    /* width:50%; */
+    margin: auto;
+    padding: 1.5rem;
+    box-sizing: border-box;
+    box-shadow : 0 0 5px rgba(0, 0, 0, 0.2)
```

Comment:
don't copy box shadow, use `box-shadow: var(--default-box-shadow)` instead.
It's available in root.